v3/src/components/map/components/inspector-panel/hide-show-menu-list.tsx

```diff
@@ -0,0 +1,43 @@
+import { MenuItem, MenuList, useToast } from "@chakra-ui/react"
+import React from "react"
+import { useDataSetContext } from "../../../../hooks/use-data-set-context"
+import { ITileModel } from "../../../../models/tiles/tile-model"
+import {isMapContentModel} from "../../models/map-content-model"
+import t from "../../../../utilities/translation/translate"
+
+interface IProps {
+  tile?: ITileModel
+}
+
+export const HideShowMenuList = ({tile}: IProps) => {
+  const data = useDataSetContext()
+  const toast = useToast()
+  const mapModel = isMapContentModel(tile?.content) ? tile?.content : undefined
```

Comment:
`mapModel` is defined here but doesn't seem to be used anywhere. Will it be utilized in the future?

Reply:
Yes, it will be used once we start implementing functionality of the buttons.